aiohttp/resolver.py

```diff
@@ -56,30 +65,41 @@ def __init__(self, loop=None, *args, **kwargs):
         self._loop = loop
         self._resolver = aiodns.DNSResolver(*args, loop=loop, **kwargs)
 
-        if not hasattr(self._resolver, 'gethostbyname'):
-            # aiodns 1.1 is not available, fallback to DNSResolver.query
-            self.resolve = self._resolve_with_query
-
     async def resolve(self, host, port=0, family=socket.AF_INET):
-        try:
-            resp = await self._resolver.gethostbyname(host, family)
-        except aiodns.error.DNSError as exc:
-            msg = exc.args[1] if len(exc.args) >= 1 else "DNS lookup failed"
-            raise OSError(msg) from exc
-        hosts = []
-        for address in resp.addresses:
-            hosts.append(
-                {'hostname': host,
-                 'host': address, 'port': port,
-                 'family': family, 'proto': 0,
-                 'flags': socket.AI_NUMERICHOST})
+        if family == 0:
+            hosts = await self._resolve(host, port, socket.AF_INET)
+            with suppress(OSError):
```

Comment:
the same as before, we can run them concurrently. Also why in this specific case the `suppress`?